modules/login/src/main/resources/META-INF/resources/login/js/registration.js

```diff
@@ -0,0 +1,401 @@
+var registration = function($) {
+    var orgLookupData = {};
+    var bInitialized = false;
```

Comment:
Probably an old habit of mine leaking in. I'll change this. 
On a related note, we should decide when prefixes are appropriate. They can be useful on loosely typed languages like javascript where it may not be obvious what the type is.